doc/internal/man3/ossl_cmp_hdr_init.pod

```diff
@@ -30,10 +30,10 @@ ossl_cmp_hdr_init
   int ossl_cmp_hdr_update_messagetime(OSSL_CMP_PKIHEADER *hdr);
   int ossl_cmp_hdr_set1_senderKID(OSSL_CMP_PKIHEADER *hdr,
                              const ASN1_OCTET_STRING *senderKID);
-  int ossl_cmp_hdr_ggeneralinfo_item_push0(OSSL_CMP_PKIHEADER *hdr,
-                                     OSSL_CMP_ITAV *itav);
-  int ossl_cmp_hdr_ggeneralinfo_items_push1(OSSL_CMP_PKIHEADER *hdr,
-                                      STACK_OF(OSSL_CMP_ITAV) *itavs);
+  int ossl_cmp_hdr_generalinfo_item_push0(OSSL_CMP_PKIHEADER *hdr,
+                                    OSSL_CMP_ITAV *itav);
+  int ossl_cmp_hdr_generalinfo_items_push1(OSSL_CMP_PKIHEADER *hdr,
+                                     STACK_OF(OSSL_CMP_ITAV) *itavs);
```

Comment:
indentation?

Reply:
fixed, also for a couple of similar cases in that internal doc file